test/helpers/cilium.go

```diff
@@ -320,18 +320,17 @@ func (c *Cilium) ReportFailed(commands ...string) {
 	wr := c.logCxt.Logger.Out
 	fmt.Fprint(wr, "StackTrace Begin\n")
 
-	//FIXME: Ginkgo PR383 add here --since option
-	res := c.Node.Exec("sudo journalctl --no-pager -u cilium")
-	fmt.Fprint(wr, res.Output())
-
-	res = c.Node.Exec("sudo cilium endpoint list")
+	res := c.Node.Exec("sudo cilium endpoint list")
 	fmt.Fprint(wr, res.Output())
 
 	for _, cmd := range commands {
 		fmt.Fprintf(wr, "Command '%s': \n", cmd)
 		res = c.Node.Exec(fmt.Sprintf("%s", cmd))
 		fmt.Fprint(wr, res.Output())
 	}
+	logCmd := "sudo journalctl --no-pager -u cilium"
```

Comment:
It seems you missed the `//FIXME: Ginkgo PR383 add here --since option`

Reply:
(+@eloycoto)
I'm not sure that we really want to dump the logs to the terminal here. One of the first things I always do with either the bash tests or these ginkgo tests is to disable printing the cilium logs so that I can see the actual error reported by the testsuite. Later, if I think the logs have something relevant I'll dig through them separately.

The issue https://github.com/onsi/ginkgo/pull/383 may still be relevant, for instance to help developers see only the relevant logs, but I'm not sure whether we need to leave a reminder in this function here. Happy to do so if reviewers think it should be here.

Reply:
On second thought, I can see that it'd be quite nice if the logCmd here that gets printed to the console would include the exact date and exact format for how to get those relevant logs. I'll keep the ginkgo issue reference when I update.

Reply:
Need to say that I would love to have the log messages on the output. 

If the log is added in the current GinkgoWritter on Junit output, you'll have all the logs, when you see the failure on Jenkins it'll be super easy to debug ;-)

Reply:
Having all logs from the agent dumped without `--since` provided will result in a huge dump of output. I'd prefer if the logs were dumped to a directory per test like we currently do in the bash-script based tests. The Jenkins console shows that a failure occurred; the logs and other data gathered and published as artifacts in the Jenkins build provide data to investigate.

Reply:
> Need to say that I would love to have the log messages on the output.
> If the log is added in the current GinkgoWritter on Junit output, you'll have all the logs, when you see the failure on Jenkins it'll be super easy to debug ;-)

Can you elaborate upon what this means? Does the GinkgoWrite write to the JUnit report for each test?

Reply:
Hey @ianvernon @joestringer 

Yes, the output of the CiliumReportFailed will be in the xml file with all description of the test. So it's not needed to have all the info in separate files, see this example: 

https://gist.github.com/eloycoto/b45690a349f39705b9484db328f47253

Regards

Reply:
Anyway, with this change, the content of the logs are not saved in the `GinkgoWriter` the command is not executed.